laalaa/apps/advisers/management/commands/ping.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from advisers.models import Organisation
+
+
+class Command(BaseCommand):
+    help = "Attempts to establish connections with vital services and raises an error if this is not possible"
+
+    def handle(self, *args, **options):
+        try:
+            # ability to connect to the db; doesn't matter if the result is True or False
+            Organisation.objects.exists()
```

Comment:
Would any of the code in https://github.com/ministryofjustice/laa-legal-adviser-api/blob/a3d04c2c865dc67e13fa3aebcb4f8eebca0fe8ca/laalaa/apps/advisers/healthchecks.py be reusable? It already implements or uses database and worker health checks.

Reply:
Yep could be useful in future. I did want to avoid using the healthcheck endpoint directly for a liveness check, but that code could indeed be reused